src/components/ApplicationJSon.vue

```diff
@@ -65,11 +80,8 @@
       :unsupported-content-msg="translations.unsupportedContent"
       :position="position"
       :document="document"
-      :on-save="saveCard"
```

Comment:
Hello, why did you remove these parameters from the <menu-list-prompt>? Aren't they beeing used?